ivy/functional/frontends/jax/random.py

```diff
@@ -0,0 +1,15 @@
+import ivy
+
+from functools import partial
+
+try:
+    from jax._src.prng import PRNGKeyArray as _PRNGKeyArray
+except ImportError:
+    from types import SimpleNamespace
+
+    random = SimpleNamespace
+
+
+def PRNGKey(seed: int) -> ivy.Array:
+    ivy.seed(seed_value=seed)
+    return random().PRNG(seed)

```

Comment:
Hi, thank you for responding to the post and I apologize for the late response. What ivy functions exactly are you referring to since I could not find them? Thank you

Reply:
As we are generating random numbers we can use the ivy random functions which are documented [here](https://lets-unify.ai/ivy/functional/ivy/random.html)